src/richie/apps/courses/templates/courses/cms/program_detail.html

```diff
@@ -0,0 +1,44 @@
+{% extends "richie/fullwidth.html" %}
+{% load cms_tags extra_tags i18n thumbnail %}
+{% block content %}
+{% spaceless %}
+<div class="program-detail">
+  <h1 class="program-detail__title">{% render_model current_page "title" %}</h1>
+
+  <div class="program-detail__cover">
+    {% get_placeholder_plugins "cover" as plugins or %}
```

Comment:
I found out it is better to have unique placeholder names among all templates. We did it on courses but not on other objects and will have to correct it eventually. So here, we should call it `program_cover`:
```suggestion
    {% get_placeholder_plugins "program_cover" as plugins or %}
```